java/test/src/main/java/io/ray/test/CrossLanguageInvocationTest.java

```diff
@@ -227,6 +307,22 @@ public static String returnInputString(String s) {
     return (byte[]) res.get();
   }
 
+  public static Object raiseException() {
```

Comment:
```suggestion
  public static Object throwException() {
```

Reply:
Please replace `raise` with `throw` in Java method names.

Reply:
OK. Thanks.